src/server/ua_server.c

```diff
@@ -143,6 +145,10 @@ void UA_Server_delete(UA_Server *server) {
     // Delete the timed work
     UA_RepeatedJobsList_deleteMembers(&server->repeatedJobs);
 
+    for(size_t i = 0; i < server->config.securityPolicies.count; ++i) {
```

Comment:
When we have a config_new() and config_delete function, this should be outside the server.
Eg if people want to reuse a config.